ui/package.json

```diff
@@ -29,7 +29,7 @@
     "react-dom": "16.4.2"
   },
   "devDependencies": {
-    "@connexta/ace": "git://github.com/connexta/ace.git#v0.0.7",
+    "@connexta/ace": "git://github.com/connexta/ace.git#20d216f6377802e2bc931331e3abf5dd8108b5c8",
```

Comment:
✏️ Seems strange to reference this by a commit number. Why not upgrade the version in ace and reference that?

Reply:
@djblue Could we get some more elaboration here on the problem that the hashes solve?

Reply:
With git hashes, I'm trying to strike the balance between the convenience snapshot versions and stability release versions.